nixos/modules/services/monitoring/netdata.nix

```diff
@@ -398,6 +449,14 @@ in {
         group = cfg.group;
         permissions = "u+rx,g+x,o-rwx";
       };
+    } // optionalAttrs (cfg.package.withNdsudo) {
+      "ndsudo" = {
+        source = "${cfg.package}/libexec/netdata/plugins.d/ndsudo.org";
+        setuid = true;
+        owner = "root";
+        group = cfg.group;
+        permissions = "u+rx,g+x,o-rwx";
```

Comment:
Not sure about these, I just copied those from other plugins. Let me know if this is too much.

Reply:
Actually, this is good, IMO. It produces the following permissions: `-r-s--x---`.